lib/fixed-points.nix

```
@@ -394,4 +394,77 @@ rec {
     fix' (self: (rattrs self) // {
       ${extenderName} = f: makeExtensibleWithCustomName extenderName (extends f rattrs);
     });
+
+  /**
+    Convert to an extending function (overlay).
+
+    `toExtension` is the `toFunction` for extending functions (a.k.a. extensions or overlays).
+    It converts a non-function or a single-argument function to an extending function,
+    while returning a double-argument function as-is.
```

```suggestion
    while returning a two-argument function as-is.
```

Maybe it's just me, but double sounds a bit like an intrinsic property, which it isn't here.